CHANGELOG.md

```diff
@@ -15,10 +15,6 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 - `handleIncomingRedirect` uses the current browser URL as a default value.
 
-The following sections document changes that have been released already:
```

Comment:
Heh, how did this happen? :smile:

Reply:
I think there was an unfortunate conflict resolution ^^